### DISCUSSION TITLE
user name

### DISCUSSION THREAD

**Discussion by BlackSparkz** [community/community#102707] [New to GitHub] [answered]
I wnt to change my user name on github 
How to change , where is option

**Comment by mustafa-tariqk** [comment, marked as answer]
![image](https://github.com/community/community/assets/58278308/f815f9bb-04c6-4442-a344-bc4bcd90f75d)

Go to Settings -> Account -> Change username -> read warning -> type new username -> If username availabe, click chage my username.

Took this from [here](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-user-account-settings/changing-your-github-username) (it's at the bottom): 

Mark this question as an answer if you found it useful! Thanks!

**Comment by ADKATT** [comment]
Hello,

Sign in to GitHub:
Sign in github account

Go to your Account Settings
In the top-right corner of the GitHub page, click on your profile picture, and then click on "Settings."
On the left sidebar, click on "Account."

Change your username
In the "Account settings" page, under the "Username" section, you'll see your current username.
Click on the "Change username" button.

New username
GitHub will ask you to enter your new desired username.
Type in the new username you want and check for its availability.

To confirm the change, you'll need to enter your password.
Click "Change username" After providing your password, click the "Change username" button to confirm.

Thanks
